src/providers/json-rpc-provider.ts

```diff
@@ -68,13 +68,17 @@ export class JsonRpcProvider extends Provider {
 
     /**
      * Query the RPC as [shown in the docs](https://docs.nearprotocol.com/docs/interaction/rpc#query)
-     * @param path Path parameter for the RPC (ex. "contract/my_token")
-     * @param data Data parameter (ex. "", "AQ4", or whatever is needed)
```

Comment:
Please add documentation to describe the new interface & behavior

Reply:
@chadoh what inft do you think is going to be useful outside of link to corresponding RPC docs?

If I added a doc in style of the docs we had before it would be something like:
```
@param params parameters for the RPC
```

which I don't think is adding value